tests/details/lending-bar.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { test } from '../fixtures';
+
+test(`Load theater: bookreader with lending bar`, async ({ lendingBar, loginPage }) => {
```

Comment:
similar to as what i mentioned in the fixtures.ts file, i think you can use `detailsPage` as fixture instead of creating a new fixture `lendingBar` for this matter